packages/angular/cli/BUILD.bazel

```diff
@@ -312,19 +307,22 @@ jasmine_node_test(
     srcs = [":angular-cli_test_lib"],
 )
 
-# @external_begin
+genrule(
+    name = "license",
+    srcs = ["//:LICENSE"],
+    outs = ["LICENSE"],
+    cmd = "cp $(execpath //:LICENSE) $@",
+)
```

Comment:
`pkg_npm` requires that included files be in the same directory, so this just copies the LICENSE file from the root.